### ISSUE TITLE
Unreliable: connection refused on all reverse proxys

### ISSUE THREAD

**Issue by DezzyTee** [jlesage/docker-nginx-proxy-manager#76]
Hi there,
everything worked fine until yesterday. Today I tried to log into my NextCloud and it didn't work. 
I tried OpenVPN, didn't work. Pretty much every reversed proxy that worked up until now doesn't work anymore.
The error log says the following:
"2020/05/09 13:53:05 [error] 1690#1690: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.192.62, server: nginxproxymanager, request: "GET /api/ HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "192.168.192.59:7818", referrer: "http://192.168.192.59:7818/""

Here's my NextCloud config in the proxy_hosts dir:
"# ------------------------------------------------------------
# nextcloud.mydomain.com
# ------------------------------------------------------------

server {
  set $forward_scheme https;
  set $server         "192.168.192.59";
  set $port           444;

  listen 8080;
listen [::]:8080;

listen 4443 ssl http2;
listen [::]:4443;

  server_name nextcloud.mydomain.com;

  # Let's Encrypt SSL
  include conf.d/include/letsencrypt-acme-challenge.conf;
  include conf.d/include/ssl-ciphers.conf;
  ssl_certificate /etc/letsencrypt/live/npm-2/fullchain.pem;
  ssl_certificate_key /etc/letsencrypt/live/npm-2/privkey.pem;

  # HSTS (ngx_http_headers_module is required) (31536000 seconds = 1 year)
  add_header Strict-Transport-Security "max-age=31536000; preload" always;

  access_log /data/logs/proxy_host-2.log proxy;

  location / {

    # Force SSL
    include conf.d/include/force-ssl.conf;

  # HSTS (ngx_http_headers_module is required) (31536000 seconds = 1 year)
  add_header Strict-Transport-Security "max-age=31536000; preload" always;

    
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection $http_connection;
    proxy_http_version 1.1;
    

    # Proxy!
    include conf.d/include/proxy.conf;
  }

  # Custom
  include /data/nginx/custom/server_proxy[.]conf;
}

The forwarded ports on my router are:
80 -> 180
443 -> 1443

I didn't change anything since yesterday. 
I don't know if it's a bug or if I screwed up the configuration.

Help would be greatly appreciated since I'm at the end of my wits right now. It's been around four hours since I started trying to fix it with no luck.

If you need any other information please let me know.

**Comment by jlesage** [comment]
Where the error message you mentioned comes from ?  From the container's log ?

**Comment by DezzyTee** [comment]
> Where the error message you mentioned comes from ? From the container's log ?

Thank you for coming back at me. It was in the error logs of manager but I was able to solve the issue. 
Turns out my ISP changed my IP so my DNS wasn't able to connect. I have no idea why I still got that error code since there shouldn't have been anything able to get through but I didn't look into it deeper after I got the original problem fixed.
